src/auth/auth.swagger.ts

```diff
@@ -1,58 +1,165 @@
+import { applyDecorators } from '@nestjs/common';
+import { ApiOperation, ApiBody, ApiResponse, ApiTags } from '@nestjs/swagger';
 import { LoginRequestDto } from './dto/login-request.dto';
 import { SignupRequestDto } from './dto/signup-request.dto';
+import { AuthResponseDto } from './dto/auth-response.dto';
 
-export const signupSwagger = {
-  apiBody: {
-    description: '회원가입 요청 DTO',
-    type: SignupRequestDto,
-  },
-  apiCreatedResponse: {
-    description: '회원가입 성공',
-    schema: {
-      example: {
-        status: 'success',
-        message: '회원가입에 성공했습니다.',
-        data: null,
-      },
-    },
+const unauthorizedExamples = {
+  TokenExpired: {
+    message: '토큰이 만료되었습니다. 다시 로그인해주세요.',
+    error: 'Unauthorized',
+    statusCode: 401,
```

Comment:
지역변수라 크게 의미는 없습니다만 가급적 UPPER_SNAKE_CASE로 명명하는 습관이 들면 좋을거같습니다

```suggestion
const UNAUTHORIZED_EXAPLES = {
  TOKEN_EXPRiED: {
    message: '토큰이 만료되었습니다. 다시 로그인해주세요.',
    error: 'Unauthorized',
    statusCode: 401,
```

Reply:
제가 미처 생각하지 못했던 부분인 것 같습니다. 그리고 코멘트를 남기지 않고 approve만 했던 이유는, 저희가 항상 코드 수정을 마친 후 대면으로 코드를 리뷰 하기 때문에 따로 코멘트를 적지 않고 approve를 했었습니다!

Reply:
넵 그럼에도 저희(서포터즈)의 승인으로 시스템상 병합을 할 수 있는 케이스를 만들지 않기 위해 comment로 남겼습니다

https://github.com/modern-agile-team/9term-main-back/pull/54#pullrequestreview-3004820905 댓글에 대한 댓글을 남기는 방법은 점 3개 클릭 - quote reply로 하시면 됩니다